wire-ios/Wire-iOS/Sources/UserInterface/Conversation/ConversationViewController+ConversationContentViewControllerDelegate.swift

```diff
@@ -152,35 +152,45 @@ extension ConversationViewController: ConversationContentViewControllerDelegate
             return
         }
 
-        let groupDetailsViewController = GroupDetailsViewController(
-            conversation: conversation,
-            userSession: userSession,
-            mainCoordinator: mainCoordinator,
-            selfProfileUIBuilder: selfProfileUIBuilder,
-            conversationCreationRepository: conversationCreationRepository,
-            isUserE2EICertifiedUseCase: userSession.isUserE2EICertifiedUseCase
-        )
-        let navigationController = UINavigationController(rootViewController: groupDetailsViewController)
-        groupDetailsViewController.presentGuestOptions(animated: false)
-        presentParticipantsViewController(navigationController, from: sourceView)
+        Task { @MainActor in
```

Comment:
We need async tasks because we query the FeatureConfigRepository.
In my opinion the feature flags should be accessible synchronously (kept in memory).

Reply:
`FeatureConfigLocalStore.fetchFeature` and `FeatureConfigLocalStore.isFeatureEnabled` could be changed fairly easily to use `context.performAndWait` instead of `await context.perform`

Reply:
> `FeatureConfigLocalStore.fetchFeature` and `FeatureConfigLocalStore.isFeatureEnabled` could be changed fairly easily to use `context.performAndWait` instead of `await context.perform`

But we don't want to block the main thread.